src/modules/previewpane/powerpreview/settings.cpp

```diff
@@ -0,0 +1,145 @@
+#include "pch.h"
+#include <common.h>
+#include "settings.h"
+#include "trace.h"
+
+namespace PowerPreviewSettings
+{
+	extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+	// Base Settinngs Class Implementation
+	FileExplorerPreviewSettings::FileExplorerPreviewSettings(bool state, std::wstring name, std::wstring description)
+	{
+        this->m_isPreviewEnabled = std::move(state);
+		this->m_name = std::move(name);
+        this->m_description = std::move(description);
+	}
+
+	FileExplorerPreviewSettings::FileExplorerPreviewSettings(bool state)
+    {
+        this->m_isPreviewEnabled = std::move(state);
```

Comment:
nit:  I just want to point out that there isn't really any advantage to the std::move cast on primitive data types.

Reply:
I replaced move with const string